From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Oleg <balbes-150@yandex.ru>
Date: Wed, 27 Oct 2021 18:56:02 +0200
Subject: [ARCHEOLOGY] u-boot usb m1 (#3221)

> X-Git-Archeology: - Revision 7789fef83f10954ab442401a86a0e5a166e5db55: https://github.com/armbian/build/commit/7789fef83f10954ab442401a86a0e5a166e5db55
> X-Git-Archeology:   Date: Wed, 27 Oct 2021 18:56:02 +0200
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: u-boot usb m1 (#3221)
> X-Git-Archeology:
> X-Git-Archeology: - Revision fd33c6545bbb4bf2383fc212dd305f3b09e47fac: https://github.com/armbian/build/commit/fd33c6545bbb4bf2383fc212dd305f3b09e47fac
> X-Git-Archeology:   Date: Wed, 27 Oct 2021 20:28:50 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Moved remaining u-boot rockchip64 mainline patches to new directory
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2b431a49151f7bc1e0e593b4b1b4fea9dfefe229: https://github.com/armbian/build/commit/2b431a49151f7bc1e0e593b4b1b4fea9dfefe229
> X-Git-Archeology:   Date: Thu, 16 Jun 2022 19:29:16 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Adjust u-boot configs for Station M1 and Station P1 (#3900)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2ca6a9381db4b875533926e0eae9d3d17f68ad06: https://github.com/armbian/build/commit/2ca6a9381db4b875533926e0eae9d3d17f68ad06
> X-Git-Archeology:   Date: Thu, 23 Jun 2022 08:30:54 +0200
> X-Git-Archeology:   From: Paolo <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip64: add rk3318-box tvbox board patch and configurations (#3921)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 4707e71e35ce43f35509ca65b1406d74d3e1c16f: https://github.com/armbian/build/commit/4707e71e35ce43f35509ca65b1406d74d3e1c16f
> X-Git-Archeology:   Date: Sun, 26 Nov 2023 13:58:04 +0100
> X-Git-Archeology:   From: Alex Shumsky <alexthreed@gmail.com>
> X-Git-Archeology:   Subject: rk3318-box: Enable uboot BTRFS support
> X-Git-Archeology:
> X-Git-Archeology: - Revision eec57ef6ce5837cf9a69b1ef03422a3ebeb9d556: https://github.com/armbian/build/commit/eec57ef6ce5837cf9a69b1ef03422a3ebeb9d556
> X-Git-Archeology:   Date: Mon, 18 Dec 2023 10:03:17 +0100
> X-Git-Archeology:   From: Alex Shumsky <alexthreed@gmail.com>
> X-Git-Archeology:   Subject: rk3318: add uboot Recovery button support
> X-Git-Archeology:
> X-Git-Archeology: - Revision 6154b98851b93e868acfa0dc5e2a9abedfce2308: https://github.com/armbian/build/commit/6154b98851b93e868acfa0dc5e2a9abedfce2308
> X-Git-Archeology:   Date: Fri, 23 Feb 2024 11:20:57 +0100
> X-Git-Archeology:   From: Alex Shumsky <alexthreed@gmail.com>
> X-Git-Archeology:   Subject: rk3318: enable uboot gpio command
> X-Git-Archeology:
> X-Git-Archeology: - Revision efee17f217e58a93e795c165e303bfd0a2a0a32a: https://github.com/armbian/build/commit/efee17f217e58a93e795c165e303bfd0a2a0a32a
> X-Git-Archeology:   Date: Mon, 22 Apr 2024 12:39:09 +0200
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip64: bump rk3318-box uboot to v2024.01
> X-Git-Archeology:
> X-Git-Archeology: - Revision e127109e2dddb7ed48a9fef0b1b60fc8d064cff0: https://github.com/armbian/build/commit/e127109e2dddb7ed48a9fef0b1b60fc8d064cff0
> X-Git-Archeology:   Date: Fri, 14 Jun 2024 00:35:08 +0200
> X-Git-Archeology:   From: Alex Shumsky <alexthreed@gmail.com>
> X-Git-Archeology:   Subject: rockchip64/rk3318-box: move stack further from base addr to allow bigger uboot image
> X-Git-Archeology:
> X-Git-Archeology: - Revision 7876017d0b77bbfefbb3d112045b32d9b50db928: https://github.com/armbian/build/commit/7876017d0b77bbfefbb3d112045b32d9b50db928
> X-Git-Archeology:   Date: Tue, 02 Jul 2024 23:31:50 +0000
> X-Git-Archeology:   From: Paolo <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: Bump rk322x-box and rk3318-box to u-boot v2024.07-rc5 (#6855)
> X-Git-Archeology:
---
 configs/rk3318-box_defconfig | 131 ++++++++++
 1 file changed, 131 insertions(+)

diff --git a/configs/rk3318-box_defconfig b/configs/rk3318-box_defconfig
new file mode 100644
index 000000000000..111111111111
--- /dev/null
+++ b/configs/rk3318-box_defconfig
@@ -0,0 +1,132 @@
+CONFIG_ARM=y
+CONFIG_SKIP_LOWLEVEL_INIT=y
+CONFIG_COUNTER_FREQUENCY=24000000
+CONFIG_ARCH_ROCKCHIP=y
+CONFIG_NR_DRAM_BANKS=1
+CONFIG_SF_DEFAULT_SPEED=20000000
+CONFIG_ENV_OFFSET=0x3F8000
+CONFIG_DEFAULT_DEVICE_TREE="rockchip/rk3318-box"
+CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_DM_RESET=y
+CONFIG_ROCKCHIP_RK3328=y
+CONFIG_ROCKCHIP_EXTERNAL_TPL=y
+CONFIG_SPL_DRIVERS_MISC=y
+CONFIG_TARGET_BOX_RK3318=y
+CONFIG_PRE_CON_BUF_ADDR=0xf200000
+CONFIG_DEBUG_UART_BASE=0xFF130000
+CONFIG_DEBUG_UART_CLOCK=24000000
+CONFIG_SYS_LOAD_ADDR=0x800800
+CONFIG_DEBUG_UART=y
+CONFIG_LOCALVERSION="-armbian"
+# CONFIG_LOCALVERSION_AUTO is not set
+CONFIG_FIT=y
+CONFIG_FIT_VERBOSE=y
+CONFIG_SPL_FIT_SIGNATURE=y
+CONFIG_SPL_LOAD_FIT=y
+CONFIG_LEGACY_IMAGE_FORMAT=y
+CONFIG_BOOTDELAY=1
+CONFIG_USE_PREBOOT=y
+CONFIG_DEFAULT_FDT_FILE="rockchip/rk3318-box.dtb"
+CONFIG_LOGLEVEL=6
+CONFIG_SILENT_CONSOLE=y
+# CONFIG_SPL_SILENT_CONSOLE is not set
+# CONFIG_TPL_SILENT_CONSOLE is not set
+# CONFIG_SYS_DEVICE_NULLDEV is not set
+# CONFIG_DISPLAY_CPUINFO is not set
+CONFIG_DISPLAY_BOARDINFO_LATE=y
+CONFIG_LAST_STAGE_INIT=y
+CONFIG_SPL_MAX_SIZE=0x40000
+CONFIG_SPL_PAD_TO=0x7f8000
+# CONFIG_SPL_RAW_IMAGE_SUPPORT is not set
+CONFIG_SPL_ATF=y
+CONFIG_SPL_ATF_NO_PLATFORM_PARAM=y
+CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_GPIO=y
+CONFIG_CMD_GPT=y
+CONFIG_CMD_MMC=y
+CONFIG_CMD_USB=y
+# CONFIG_CMD_SETEXPR is not set
+CONFIG_CMD_TIME=y
+CONFIG_CMD_BTRFS=y
+CONFIG_SPL_OF_CONTROL=y
+CONFIG_TPL_OF_CONTROL=y
+CONFIG_OF_SPL_REMOVE_PROPS="clock-names interrupt-parent assigned-clocks assigned-clock-rates assigned-clock-parents"
+CONFIG_TPL_OF_PLATDATA=y
+CONFIG_ENV_IS_IN_MMC=y
+CONFIG_SYS_RELOC_GD_ENV_ADDR=y
+CONFIG_SYS_MMC_ENV_DEV=1
+CONFIG_TPL_DM=y
+CONFIG_SPL_DM_SEQ_ALIAS=y
+CONFIG_REGMAP=y
+CONFIG_SPL_REGMAP=y
+CONFIG_TPL_REGMAP=y
+CONFIG_SYSCON=y
+CONFIG_SPL_SYSCON=y
+CONFIG_TPL_SYSCON=y
+CONFIG_BUTTON=y
+CONFIG_BUTTON_ADC=y
+CONFIG_BUTTON_GPIO=y
+CONFIG_CLK=y
+CONFIG_SPL_CLK=y
+CONFIG_ROCKCHIP_GPIO=y
+CONFIG_SYS_I2C_ROCKCHIP=y
+CONFIG_MMC_DW=y
+CONFIG_MMC_DW_ROCKCHIP=y
+CONFIG_DM_MDIO=y
+CONFIG_DM_ETH_PHY=y
+CONFIG_PHY_GIGE=y
+CONFIG_ETH_DESIGNWARE=y
+CONFIG_RGMII=y
+CONFIG_MII=y
+CONFIG_RMII=y
+CONFIG_GMAC_ROCKCHIP=y
+CONFIG_NOP_PHY=y
+CONFIG_PHY_ROCKCHIP_INNO_HDMI=y
+CONFIG_PHY_ROCKCHIP_INNO_USB2=y
+CONFIG_PINCTRL=y
+CONFIG_SPL_PINCTRL=y
+CONFIG_DM_PMIC=y
+# CONFIG_SPL_DM_PMIC is not set
+CONFIG_PMIC_RK8XX=y
+CONFIG_REGULATOR_PWM=y
+CONFIG_DM_REGULATOR_FIXED=y
+CONFIG_REGULATOR_RK8XX=y
+CONFIG_PWM_ROCKCHIP=y
+CONFIG_RAM=y
+CONFIG_SPL_RAM=y
+CONFIG_TPL_RAM=y
+CONFIG_DM_RNG=y
+CONFIG_RNG_ROCKCHIP=y
+CONFIG_BAUDRATE=1500000
+CONFIG_DEBUG_UART_SHIFT=2
+CONFIG_DEBUG_UART_ANNOUNCE=y
+CONFIG_SYS_NS16550_MEM32=y
+CONFIG_SYSINFO=y
+CONFIG_SYSRESET=y
+# CONFIG_TPL_SYSRESET is not set
+CONFIG_USB=y
+CONFIG_USB_XHCI_HCD=y
+CONFIG_USB_XHCI_DWC3=y
+CONFIG_USB_EHCI_HCD=y
+CONFIG_USB_EHCI_GENERIC=y
+CONFIG_USB_OHCI_HCD=y
+CONFIG_USB_OHCI_GENERIC=y
+CONFIG_USB_DWC2=y
+CONFIG_USB_DWC3=y
+CONFIG_USB_DWC3_GENERIC=y
+CONFIG_USB_KEYBOARD=y
+CONFIG_USB_KEYBOARD_FN_KEYS=y
+CONFIG_VIDEO=y
+# CONFIG_BACKLIGHT_PWM is not set
+CONFIG_VIDEO_ANSI=y
+# CONFIG_PANEL is not set
+CONFIG_DISPLAY=y
+CONFIG_VIDEO_ROCKCHIP=y
+CONFIG_DISPLAY_ROCKCHIP_HDMI=y
+CONFIG_VIDEO_SIMPLE=y
+CONFIG_SPL_TINY_MEMSET=y
+CONFIG_TPL_TINY_MEMSET=y
+CONFIG_ERRNO_STR=y
+# CONFIG_EFI_LOADER is not set
+CONFIG_BOARD_RNG_SEED=y
+CONFIG_CMD_RNG=y
-- 
Armbian

